Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderBuilderUtility.cpp

```diff
@@ -460,6 +460,13 @@ namespace AZ
                 {
                     platformId = AzFramework::PlatformId::PC;
                 }
+                else if (platformIdentifier == "linux")
+                {
+#pragma push_macro("LINUX")
+#undef LINUX
```

Comment:
This looks annoying. I wonder if the LINUX macro/enum name can be changed so that the code does not need to push_macro each time?

Reply:
I've renamed this to `LINUX_ID`, because it failed in the validator as well.